### PR TITLE
Add write prescriptions

### PR DESCRIPTION
**Pull request by raaidarshad**
Adds the write_prescriptions function and tests. Makes a few other small updates, like bumping the ata-db-models version, consolidation of some test helpers, adds a logging helper, and a tweak to the process function.

### CODE REVIEW

File: ata_pipeline1/helpers/logging.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import logging
+import logging.config
+
+# To use this logging config in another module:
+# >>> from ata_pipeline0.helpers.logging import logging
```
Comment (duynguyen158) [inline ata_pipeline1/helpers/logging.py:5]: ⛏ I'll change this to `ata_pipeline1` haha

File: tests/_helpers.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from collections.abc import Generator
```
Comment (duynguyen158) [inline tests/_helpers.py:1]: ❓ Not important at all & just out of curiosity: Do we need the underscore in the module name? Tried Googling it and didn't come across anything.

File: tests/test_write_prescriptions.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+def test_write_prescriptions(prescriptions_as_df: pd.DataFrame, engine: Engine, session_factory: sessionmaker) -> None:
+    with create_and_drop_tables(engine):
+        rows_written = write_prescriptions(df=prescriptions_as_df, session_factory=session_factory)
+        assert rows_written == prescriptions_as_df.shape[0]
+        with session_factory.begin() as session:  # type: ignore
+            statement = select(Prescription)
+            results = session.execute(statement)
+            data = [prescription[0].dict() for prescription in results]
+    print(prescriptions_as_df)
```
Comment (duynguyen158) [inline tests/test_write_prescriptions.py:37]: ⛏ Assuming you want to delete this before merging.

File: tests/test_write_prescriptions.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    # order the columns the same way as prescriptions_as_df
+    from_db = pd.DataFrame(data=data)[["user_id", "site_name", "prescribe", "last_updated"]]
+    print(from_db)
```
Comment (duynguyen158) [inline tests/test_write_prescriptions.py:40]: ⛏ Same here.

**Review by duynguyen158 — APPROVED** [review]
LGTM! Left a few comments that aren't terribly important.

File: tests/_helpers.py (hunk 2 above)
Comment (raaidarshad) [reply tests/_helpers.py:1]: We don't, can rename

File: tests/test_write_prescriptions.py (hunk 3 above)
Comment (raaidarshad) [reply tests/test_write_prescriptions.py:37]: hah yup ty

**Pull request merged**